ci-operator/jobs/openshift/aws-load-balancer-operator/openshift-aws-load-balancer-operator-main-presubmits.yaml

```diff
@@ -119,6 +119,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-operator,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
+    - ^main-
+    cluster: build01
+    context: ci/prow/e2e-aws-proxy-operator
+    decorate: true
+    labels:
+      ci-operator.openshift.io/cloud: aws
+      ci-operator.openshift.io/cloud-cluster-profile: aws
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-aws-load-balancer-operator-main-e2e-aws-proxy-operator
+    optional: true
+    rerun_command: /test e2e-aws-proxy-operator
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --lease-server-credentials-file=/etc/boskos/credentials
+        - --report-credentials-file=/etc/report/credentials
+        - --secret-dir=/secrets/ci-pull-credentials
+        - --secret-dir=/usr/local/e2e-aws-proxy-operator-cluster-profile
+        - --target=e2e-aws-proxy-operator
+        command:
+        - ci-operator
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
```

Comment:
Doesn't this need a special volume mount too?

Reply:
For trusted CA configmap? If yes - unfortunately the existing ci-operator's step (`ipi-aws-pre-proxy`) doesn't setup additional trusted CAs. Something to be added as a new ci-operator's step in the future but that would take quite some time.